container/testdata/test-deploy.yaml

```diff
@@ -0,0 +1,661 @@
+---
```

Comment:
could you name this file `test-deploy.cassette.yaml` so that it is clear that it is a cassette